content/en/security/vulnerabilities/_index.md

```diff
@@ -16,48 +14,58 @@ further_reading:
 <div class="alert alert-warning">Cloud Security Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
 {{< /site-region >}}
 
-<div class="alert alert-info">CSM Vulnerabilities is in beta.</div>
-
 ## Overview
 
-Cloud Security Management Vulnerabilities (CSM Vulnerabilities) combines infrastructure observability with security insights to detect, prioritize, and manage vulnerabilities in your organization. It scans for vulnerabilities in the [container images][2] and [hosts][4] in your infrastructure.
+Cloud Security Management Vulnerabilities (CSM Vulnerabilities) helps you proactively secure your cloud infrastructure by detecting, prioritizing, and managing vulnerabilities across your container images and hosts. It leverages deep observability context and industry insights to help you remediate vulnerabilities that are most important to you at a given point in time. Details are shown on the CSM [Vulnerability Explorer][1] identifying the context-based severity, affected containers and hosts, and remediation instructions to solve the most impacted infrastructure.
+
+**Note**: If you're looking for vulnerability management for your application libraries and custom application code, see [Application Vulnerability Management][5].
+
+## Explore vulnerabilities
+The CSM Vulnerability Explorer shows a complete list of vulnerabilities detected across your infrastructure, ordering them based on their severity, and offering grouping and filtering capabilities so you can investigate and prioritize problems. 
 
-## Review the health of your container images
-CSM Vulnerabilities offers vulnerabilities in the container images on your hosts.
+(update screen shot when beta tags are removed)
 
-- View your container images on the [container images][2] page. 
-- Observe the number of vulnerabilities that exist in the container images (column name `VULNERABILITIES`).
+{{< img src="security/vulnerabilities/CSM_Vulnerabilities.png" alt="The CSM Vulnerability page sorting by unique vulnerabilities" width="100%">}}
 
-{{< img src="security/infrastructure_vulnerabilities/container_images_tab.png" alt="The Container Images tab highlighting vulnerabilities and container column sort feature" width="100%">}}
+## Manage container vulnerabilities
 
-View additional details about any vulnerability by clicking the container image and reviewing the **Vulnerabilities** tab:
+View your container images on the [container images][2] page. 
+Observe the number of vulnerabilities that exist in the container images (column name **vulnerabilities**). Additionally, sort by **source**, **image tag**, **repo digest**, and more.
```

Comment:
I feel like the first two sentences here should be combined, in a similar way to what you currently have written for the Explore Vulnerabilities section.

Reply:
Noted.